project_stage_closed/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Project Task Stage Closed",
+
+    'summary': """
+        Make the Closed flag on Task Stages available without
+        installing sale_service""",
```

Comment:
I don't know. It looks find on screen: 
![image](https://cloud.githubusercontent.com/assets/692075/11839178/1109d886-a3ec-11e5-9724-40626f3f24e9.png)